lib/db/options.js

```diff
@@ -48,4 +74,22 @@ const set = (conn, database, databaseOptions, params) => {
   ).then(httpBody);
 };
 
-module.exports = { get, set };
+const getDbConfigProperties = (conn, params) => {
```

Comment:
shorter more consistent name? `getAvailable`?

Reply:
how about getMetadata?  It's different from get() and getAll() where it returns all properties with default values plus labels and description.